pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -315,6 +315,20 @@ const (
 	DefaultNodeVolumeGP3IOPS = 3000
 )
 
+type IPFamily string
+
+const (
+	// IPV4Family defines an ip family of v4 to be used when creating a new VPC.
+	IPV4Family IPFamily = "ipv4"
+	// IPV6Family defines an ip family of v6 to be used when creating a new VPC.
+	IPV6Family IPFamily = "ipv6"
```

Comment:
I agree that it would be more readable, however, CF expects lowercase input values - see https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-eks/pull/56/files#diff-4ba214c0c8b1426b24d21b0ed5bb2beb7fdc52f5e6899c6e77b2fd1e04e731e0R45

Reply:
CF is an implementation detail while this is the UX for this feature. We don't need to follow the API schema when designing the UX.

Reply:
That's true, I think we were taking `ipv4` for granted until now but I agree with you that capitalising it would make a cleaner UX. 

Your other comments are also good catches! Would you like to open a quick PR to add your suggestions?

Reply:
Why would we allow multiple values for this setting @cPu1?

Reply:
Ah, we won't. That example `ipFamilies` snippet was taken from the Kubernetes `Service` resource. It was only to show that it uses the uppercase version.

Reply:
Oh, okay. :D Thanks. :DD

Reply:
>Would you like to open a quick PR to add your suggestions?

Looks like Gergely is working on it.